src/_help/bump-cli.md

```diff
@@ -28,164 +35,205 @@ Or, with Yarn via
 yarn global add bump-cli
 ```
 
-## Add Bump CLI to your node project
+### Add Bump.sh to your node project
 
-As our CLI is a node package, you can easily embed it to your project by adding the package to your `package.json` file, either via NPM
+As our CLI is a node package, you can easily embed it to your project by adding the package to your `package.json` file, either with NPM
 
 ```shell
 npm install --save-dev bump-cli
 ```
 
-Or via Yarn
+Or with Yarn via
 
 ```shell
 yarn add --dev bump-cli
 ```
 
-You can then use any Bump CLI commands via `npm exec` as such
+You can then use any Bump.sh commands with `npx` (same as `npm exec`)
 
 ```shell
-npm exec -- bump --help
+npx bump --help
 ```
 
-## How should I do if I'm not using Node ?
+### How should I do if I'm not using npm ?
 
-Unfortunately, at the moment we only support the Node environment. However we plan to distribute universal binaries in the most common package managers very soon. In the meantime, you can download a standalone package directly from the [latest Github release](https://github.com/bump-sh/cli/releases) assets or you can push your documentation using [our API](https://developers.bump.sh/).
+Unfortunately, at the moment we only support the Node environment. However, you can download a standalone package directly from the [latest Github release](https://github.com/bump-sh/cli/releases) assets which you can run as a standalone binary. Or you can push your documentation using [our API](https://developers.bump.sh/) (advanced usage only).
 
-## Using Bump CLI
+## Usage
 
-To list all the available commands, just type `bump` in your command line environment. You can get some help anytime by using `--help` on any command. Example: `bump deploy --help` .
+To list all the available commands, just type `bump` in your command line environment.
 
-## Preview a file
+```shell
+$ bump --help
+The Bump.sh CLI is used to interact with your API documentation hosted on Bump.sh by using the API of developers.bump.sh
```

Comment:
```suggestion
The Bump.sh CLI lets you interact with your API documentation hosted on Bump.sh by using our API (read the docs at [https://developers.bump.sh](https://developers.bump.sh)).
```

Reply:
This text is the output text of the help command, there's no need to change it here (or we should change the command output too)